app/api/resources/user.py

```diff
@@ -16,6 +16,8 @@
 from app.api.models.user import *
 from app.api.dao.user import UserDAO
 from app.api.resources.common import auth_header_parser
+from smtplib import *
```

Comment:
Instead of star importing, I would suggest writing names of all functions/modules required.

Reply:
I agree with kapil here many linter checkers discourage wildcard imports.